packages/editor-ui/src/components/RunDataAi/RunDataAiContent.vue

```diff
@@ -85,6 +90,15 @@ const runMeta = computed(() => {
 	}
 	return extractRunMeta(outputRun.value);
 });
+
+const executionRunData = computed(() => {
+	return workflowsStore.getWorkflowExecution?.data?.resultData?.runData;
+});
+
+const outputError = computed(() => {
+	return executionRunData.value?.[props.inputData.node][props.inputData.runIndex]
```

Comment:
Nit: In case `executionRunData.value` is not defined the resulting value will be `undefined as NodeError`.